com.unity.build-report-inspector/Editor/BuildReportInspector/Mobile/Tools/Utilities.cs

```diff
@@ -0,0 +1,36 @@
+#if UNITY_EDITOR
+using System;
+using System.Diagnostics;
+using System.IO;
+
+namespace Unity.Mobile.BuildReport.Tools
```

Comment:
This namespace is not consistent to others. On purpose?

Reply:
Will fix

Reply:
Fixed